packages/assets/schemas/visualResponses.schema.json

```diff
@@ -18,7 +18,7 @@
             "componentProperty": {
                 "description": "The property name driving this visualization",
                 "type": "string",
-                "enum": ["xAxis", "yAxis", "button", "state"]
+                "enum": ["x-axis", "y-axis", "button", "state"]
```

Comment:
With the new name `componentProperty`, `"x-axis"` and `"y-axis"` both make sense to me - it's easy to think of a physical thumbstick component as being composed of an `"x-axis"` property and a `"y-axis"` property.

However, a physical A button component is not itself composed of a `"button"` property or a `"state"` property.  It's also not jumping out at me when to use `"button"` and when to use `"state"`. (is it float value vs. bool value?)

Could both `"button"` and `"state"` be replaced with just `"value"`?  Then all three would feel like properties that a given component can have.

Reply:
I agree that `"button"` now seems a bit strange.  However it isn't the same as `"state"`.  `"state"` is the combined opinion of whether a component is pressed, touched, or not being interacted with.  Specifically, when an axis value is greater than the threshold of the deadzone, the component is considered "touched" whether or not it has an associated `GamepadButton` with the `touched` attribute set to `"true"`.

That said, I completely agree that `"button"` is confusing, and I'm absolutely open to renaming it.  `"value"` seems like it'd be a bit confusing since `x-axis` and `y-axis` are also values.... Any suggestions?

Either way, i'm inclined to do that in a different PR to get this one landed.  Objections?

Reply:
Sounds good - let’s carve this off as a new issue.